### ISSUE TITLE
ColorPicker Accessibility

### ISSUE THREAD

**Issue by InaGlushkova** [telerik/kendo-react#953]
ColorPicker accessibility enhancement.
Spets to reproduce the behavior:

1. Open ColorPicker dropdown
2. Navigate to some color either with a mouse or keyboard arrows

 _Expected results:_

There should be a tooltip on the hover/focus state of each color in the color picker palette since some users don't see the color's normally or are totally color blind, this tooltip would provide them a textual representation of each color inside the tooltip component.

_Actual results:_

There is no tooltip presented for each color while the user navigates through the color palette

Requested under:
1521338
